<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCannyDialog</class>
 <widget class="QDialog" name="QCannyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Canny边缘检测</string>
  </property>
  <property name="windowIcon">
   <iconset resource="picdealsrc.qrc">
    <normaloff>:/resources/PicDeal.ico</normaloff>:/resources/PicDeal.ico</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>参数A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>参数B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>算子内核</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numA">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numC">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="predeal">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>是否进行预处理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BNOK">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BNCAL">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="picdealsrc.qrc"/>
 </resources>
 <connections/>
</ui>
